Yoast/wordpress-seo
# Move notice paginated comments to health check

## Summary


This PR can be summarized in the following changelog entry:

* Moved the notice from Paginated commments from the dashboard to WordPress' Health Check

## Relevant technical choices:

* Combined the upgrade routine introduced in https://github.com/Yoast/wordpress-seo/pull/14019 with the routine made by this pull. Because of both should be done on 12.8

## Test instructions

This PR can be tested by following these steps:

* Go to Settings - Discussion and check the checkbox for paginating comments:
![Discussion_Settings_‹_Local_WordPress_Dev_—_WordPress](https://user-images.githubusercontent.com/325040/65487499-4d579180-dea7-11e9-8b62-0916c778e6d2.png)
* Go to the site health page: Tools - Site Health and see a check being presented:
![Site_Health_Status_‹_Local_WordPress_Dev_—_WordPress](https://user-images.githubusercontent.com/325040/65487579-71b36e00-dea7-11e9-99bb-d3be044925bd.png)
* Uncheck the checkbox and verify the message being gone.

## UI changes

## Documentation

## Quality assurance


Fixes #12898
